internal/config/config_test.go

```diff
@@ -176,3 +232,47 @@ func TestConvertHCLtoTOML(t *testing.T) {
   user = "lab-testing"
 `, string(cfgData))
 }
+
+func TokenTest(t *testing.T) {
```

Comment:
Tests need to start with be in the form `TestXxx` to run properly. 

Ref: https://golang.org/pkg/testing/

Reply:
Here and for `TokenLoadTest`

Reply:
Ah thanks -- testing and fixing.

Reply:
Okay fixes applied.  Thanks for catching that and the pointer to testing.